src/lua/api-gateway/tracking/validator/blockingRulesValidator.lua

```diff
@@ -55,7 +55,7 @@ function _M:validate_blocking_rules(config_obj)
         return ngx.HTTP_OK, ""
     end
 
-    ngx.var.blocked_by = math.floor(tonumber(blocking_rule.id)/100000)
+    ngx.var.blocked_by = tonumber(blocking_rule.id)
```

Comment:
There was a reason to strip the last 5 digits from the policy ID (which actually are a timestamp referencer).

Reply:
@constantincristian, I thought so myself. However, if your ID is 250. Floor of 0,0025 will give you 0. Hence, the issue in the logs...

Reply:
Converting to string brings us back to the same issue. `print(blocking:sub(1, #blocking - 5))` Only works for strings with length greater than 5.